<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDialog</class>
 <widget class="QDialog" name="NewDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建通讯</string>
  </property>
  <property name="windowIcon">
   <iconset resource="comm.qrc">
    <normaloff>:/res/res/TCP.ico</normaloff>:/res/res/TCP.ico</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>271</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="newTabBASE">
    <attribute name="title">
     <string>通讯基本测试</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>111</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>通讯类型选择</string>
     </property>
     <widget class="QRadioButton" name="radioButtonTCPS">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TCP Server</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonTCPC">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TCP Client</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonUDP">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>UDP</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonMulticast">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>组播</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonSerial">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>串口</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>本地IP</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxLIP">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>0.0.0.0</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>本地端口</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLPORT">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>2404</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRPORT">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>2404</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxRIP">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>0.0.0.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>255.255.255.255</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>远端IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>远端端口</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>多播参数</string>
     </property>
     <widget class="QRadioButton" name="radioButtonMulticastS">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>多播源</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonMulticastC">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>加入多播</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMPORT">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>1888</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>多播端口</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>311</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>串口参数</string>
     </property>
     <widget class="QComboBox" name="comboBoxCOM_NUM">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>串口号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>波特率</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxCOM_BAUD">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentText">
       <string>110</string>
      </property>
      <item>
       <property name="text">
        <string>110</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>300</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>56000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>128000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>256000</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>停止位</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxCOM_STOPBIT">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1位</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.5位</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2位</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>数据位</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxCOM_DATABIT">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>8位</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7位</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>校验位</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxCOM_CHECK">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>无校验</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>奇校验</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>偶校验</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mark校验</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Space校验</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="newTabPING">
    <attribute name="title">
     <string>PING</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="newTabSSH">
    <property name="toolTip">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <attribute name="title">
     <string>SSH</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <zorder>pushButtonOK</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <resources>
  <include location="comm.qrc"/>
 </resources>
 <connections/>
</ui>
